client/asset/interface.go

```diff
@@ -380,6 +381,11 @@ type Wallet interface {
 	Lock() error
 	// Locked will be true if the wallet is currently locked.
 	Locked() bool
+	// LockTimeExpired returns true if the specified locktime has expired,
+	// making it possible to redeem the locked coins.
+	LockTimeExpired(ctx context.Context, lockTime time.Time) (bool, error)
+	// SendTransaction broadcasts a raw transaction, returning its coin ID.
+	SendTransaction(rawTx []byte) ([]byte, error)
```

Comment:
Just because the this functionality is already required by `AuditContract` implementations, so we know it's going to be trivial.  I'll move it into `Bonder` though.